### DISCUSSION TITLE
after SAML auth, never asks login again

### DISCUSSION THREAD

**Discussion by nerdCopter** [yuezk/GlobalProtect-openconnect#168] [Q&A] [answered]
product is great and works better than official global-protect on Linux, TY!!

After my SAML auth, it never asks to login/auth again.  While i personally may like this, i'm sure it's not considered secure to be able to reconnect to VPN days or weeks later without asking for re-auth.   Is this this products failure or the Firewall configuration at fault?

**Comment by yuezk** [comment, marked as answer]
The popup window is a webview that could save the login status. You can try to choose not to remember the login status when performing the authentication. For example, I'm using Microsoft as the SAML IDP, and I can choose `No` for the below dialog.

<img width="420" alt="image" src="https://user-images.githubusercontent.com/3297602/182344048-5eb17961-c9a3-40db-8ca5-b238ffab10b5.png">

Check your IDP to see if there is a similar option.

**Reply by nerdCopter** [reply]
Thank you, I found the `reset` menu-item, and indeed this exists.  I was accustomed to Official PaloAlto Global Protect which forces 2FA every login even if i answer "yes". (at least on our VPN implementation)
